xrpl/models/amounts/__init__.py

```diff
@@ -14,6 +16,8 @@
 from xrpl.models.amounts.issued_currency_amount import IssuedCurrencyAmount
 from xrpl.models.amounts.mpt_amount import MPTAmount
 
+ClawbackAmount = Union[IssuedCurrencyAmount, MPTAmount]
+
```

Comment:
I would create a new module for this named `clawback_amount.py` similar to how it's done for `Amount` in `xrpl/models/amounts/amount.py`

Reply:
fixed in 9144d2c5